agent/websockets/shim.go

```diff
@@ -272,12 +289,14 @@ func ShimBody(shimPath string) (func(resp *http.Response) error, error) {
 type sessionMessage struct {
 	ID      string      `json:"id,omitempty"`
 	Message interface{} `json:"msg,omitempty"`
+	Version int         `json:"v,omitempty"`
 }
 
 func createShimChannel(ctx context.Context, host, shimPath string, rewriteHost bool, openWebsocketWrapper func(http.Handler) http.Handler) http.Handler {
 	var connections sync.Map
 	var sessionCount uint64
 	mux := http.NewServeMux()
+	var protocolVersion int
```

Comment:
This variable is dead code now and should be removed.

Reply:
Merge issues as mentioned above.